lib/config.py

```diff
@@ -145,7 +141,7 @@ def discover_packages():
     versions_repo_url = config.get('build_versions_repository_url')
     versions_repo_name = os.path.basename(os.path.splitext(versions_repo_url)[0])
     build_versions_repo_dir = os.path.join(
-        config.get('build_versions_repo_dir'),
+        config.get('work_dir'),
```

Comment:
Make sense but, with the current code structure, this chance will introduce a circular dependency which will require a code refactor. Out of the scope of this patch set, if required should be done in another patch.